tokio/src/runtime/builder.rs

```diff
@@ -449,7 +482,7 @@ impl fmt::Debug for Builder {
         fmt.debug_struct("Builder")
             .field("kind", &self.kind)
             .field("num_threads", &self.num_threads)
-            .field("thread_name", &self.thread_name)
+            .field("thread_name_fn", &"<dyn Fn()->String+Send+Sync+'static>")
```

Comment:
nit:
```suggestion
            .field("thread_name_fn", &"<dyn Fn() -> String + Send + Sync + 'static>")
```
(it probably would be fine if this was just `...`, but this is definitely a little nicer)